### DISCUSSION TITLE
Select Object with mouse click but lock movement of object.

### DISCUSSION THREAD

**Discussion by Seda-cpu** [fabricjs/fabric.js#8597] [Q&A] [answered]
Hi friends;

I want to select an object but lock movement, orientation and resizing of object?

I am writing an gui with fabricjs and some of my object only for giving info to user. So User should not move the object. just click and get info about it. 

I tried those properties but when ı activate selectable property it can be moveable.
`ownCaching: false,
        selectable: true,
        hasBorders: false,
        hasRotatingPoint: false,
        hasControls: false,
        objectCaching: false,
        lockMovementX: false,
        lockMovementY: false`

How can I do this?

**Comment by melchiar** [comment, marked as answer]
the properties `lockScalingX`, `lockScalingY`, `lockRotation`, `lockMovementX` and `lockMovementY` should give you exactly what you want. If it's not working please share a fiddle showing the issue.

Just to clarify, these props will only disable the transformations done with the canvas controls (it won't prevent transformations done programmatically)

**Reply by Seda-cpu** [reply]
Thank you so much. That's exactly what I want.
